Code/Framework/AzToolsFramework/Tests/AssetFileInfoListComparison.cpp

```diff
@@ -198,6 +213,21 @@ namespace UnitTest
 
         }
 
+        void SetAssetFolder(const AZStd::string& folder)
+        {
+            const AZStd::string cacheProjectRootFolder = folder;
+            const AZStd::string cacheRootFolder = cacheProjectRootFolder + AZ_CORRECT_FILESYSTEM_SEPARATOR + AzFramework::PlatformHelper::GetPlatformName(AzFramework::PlatformId::PC);
```

Comment:
Also for path manipulation, AZ::IO::Path should be used instead of manipulating the string directly.

Reply:
This has been done, resolving.